docs/contributing.md

```diff
@@ -0,0 +1,103 @@
+# Contribution guidelines for Prem-Operator
+
+## Welcome
+
+We are excited to have you contribute to the Prem-Operator project!
+We welcome all contributions, including bug fixes, feature requests, documentation improvements, and more.
+
+## Making Contributions
+
+### Bug Fixes
+
+- If you've found a bug and have a fix, simply **open a Pull Request (PR)** with your changes.
+- Ensure your PR description clearly describes the bug and fix. Include any relevant issue numbers.
+
+### New Features
+
+- For new features, we ask that you **open an issue first** to discuss your ideas with the project maintainers.
```

Comment:
I don't mind if people make a PR first. I think the main thing is that they understand that without buy-in it's possible that we won't review the code, but just dismiss it on principle.

Reply:
Done